ui/apps/platform/src/Containers/MainPage/MainPage.tsx

```diff
@@ -51,6 +63,17 @@ function MainPage(): ReactElement {
     const { isFeatureFlagEnabled, isLoadingFeatureFlags } = useFeatureFlags();
     const { hasReadAccess, hasReadWriteAccess, isLoadingPermissions } = usePermissions();
 
+    // Check for clusters under management
+    // if none, and user can admin Clusters, redirect to clusters section
+    // (only applicable in Cloud Services version)
+    useQuery<ClusterCountResponse>(CLUSTER_COUNT, {
```

Comment:
Another thought, I wonder if it would be worth adding some text either in the UI or on the docs page that tells the user they need to refresh the ACS UI once a cluster is added. Since Apollo caches the GQL query results, this call will continue to return `0` every time the user navigates to a new page, even after a cluster is added.

Reply:
^ @lmaynard88 
Is Dave's suggestion something you can add to the docs?